elastalert/elastalert.py

```diff
@@ -994,6 +1010,36 @@ def start(self):
             sleep_duration = total_seconds(next_run - datetime.datetime.utcnow())
             self.sleep_for(sleep_duration)
 
+    def wait_until_responsive(self, timeout, clock=timeit.default_timer):
+        """Wait until ElasticSearch becomes responsive (or too much time passes)."""
+
+        # Elapsed time is a floating point number of seconds.
+        timeout = timeout.total_seconds()
+
+        # Periodically poll ElasticSearch.  Keep going until ElasticSearch is
+        # responsive *and* the writeback index exists.
+        ref = clock()
+        while (clock() - ref) < timeout:
+            try:
+                if self.writeback_es.indices.exists(self.writeback_index):
```

Comment:
I see. That's very reasonable. 

I would like to lower the default timeout in that case. If someone had incorrectly configured the index, I expect that they would be confused by the lack of output and might terminate the program before the timeout expired.

Maybe something like 1 second, which would allow the initial check to happen but would not even mislead people as to why nothing was happening. Thoughts?